frontend/plan/src/components/selector/CourseList.js

```diff
@@ -31,37 +31,76 @@ const courseSort = (courses, sortMode) => {
     return sorted;
 };
 
-export default function CourseList({ courses, getCourse, sortMode }) {
-    return (
-        <div className="scroll-container">
-            <div style={{
-                display: "flex", flexDirection: "row", paddingBottom: "1em", paddingLeft: "2em",
-            }}
-            >
-                <div className="header" style={{ overflow: "hidden", width: "60%" }}>
+export default class CourseList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.listRef = React.createRef();
+    }
+
+
+    componentDidMount() {
```

Comment:
We only want to use scrollTop from selector state only if it comes from the "Back" button (clearCourse redux action I think) right? Or else you would be skipping courses for a totally new query.

Reply:
great point -- i'll look into fixing this.

Reply:
added useEffect which sets scrollTop to 0 when course changes, not sure if that works tho

Reply:
it no work :(

Reply:
moved to redux state